samples/nrf9160/modem_shell/overlay-esp-wifi.conf

```diff
@@ -0,0 +1,25 @@
+# ESP 8266 WIFI
+CONFIG_WIFI=y
+CONFIG_WIFI_LOG_LEVEL_DBG=y #optional
```

Comment:
Should default level be debug?

Reply:
This prints quite little information. The main information is info about successful initialization (failure would be LOG_ERR) and scanning results.
This is rather nice to have with current setups with external WiFi devices and when you enable WiFi, that information is nice to have.
Decided to keep DBG level logging in the overlay for now.